lesson03/assignment/javascript.js

```diff
@@ -0,0 +1,62 @@
+
+//Setting up jQuery function
+$(function () {
+
+//JASON data for page
+const cardInfo = '[{"name":"Paolo Maldini","jobTitle":"Front End Developer","company":"Web Weavers","experience":"3 years","codeLanguages":["HTML", " CSS", " JavaScript", " Node", " Express"],"school":"University of Washington","major":"Marketing","email":"paolo@example.com","linkedInUrl":"paolo.linkedinprofile.com"},{"name":"Barbara Bonansea","jobTitle":"Software Engineer","company":"Excellence in the Cloud","experience":"12 years","codeLanguages": [" HTML", " JavaScript", " C", " Go"],"school":"University of Southern California","major":"Computer Science","email":"barbara@example.com","linkedInUrl":"barbara.linkedinprofile.com"},{"name":"Javier Hernandez","jobTitle":"User Experience Engineer","company":"Web Sites and More","experience":"5 years","codeLanguages": [" HTML", " CSS"],"school":"Seattle University","major":"Performing Arts","email":"javier@example.com","linkedInUrl":"javier.linkedinprofile.com"},{"name":"Maribel Dominguez","jobTitle":"Front End Engineer","company":"Bits and Bytes","experience":"6 years","codeLanguages": [" HTML", " CSS", " JavaScript", " React", " Vue", " Redux"],"school":"University of Washington","major":"Mechanical Engineering","email":"maribel@example.com","linkedInUrl":"maribel.linkedinprofile.com"}]';
+const card = JSON.parse(cardInfo);
+
+//building out html page
+const container = document.querySelector('.template-hook')
+
+// Loop through all elements of the list
+let cardData = card.forEach(function(el){
+
+  //Build html flexbox/grid element
+  let div = document.createElement('div');
+  div.className = 'main';
+  container.append(div);
+
+  let div2 = document.createElement('div');
+  div2.className = "box1";
+  div.append(div2);
```

Comment:
Same comment here as the previous assignment- this code will be easier to maintain with meaningful names- "personCard" rather than "div", "intro" rather than "div2", etc.